haqq/assetlist.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "haqq",
+  "assets": [
+    {
+      "description": "The native EVM, governance and staking token of the Haqq Network",
+      "denom_units": [
+        {
+          "denom": "aISLM",
+          "exponent": 0
+        },
+        {
+          "denom": "ISLM",
+          "exponent": 18
+        }
+      ],
+      "base": "aISLM",
+      "name": "Islamic Coin",
+      "display": "islm",
```

Comment:
the display denom must be listed in the denom_units array and is case sensitive. Please either correct display to ISLM (make uppercase), or the latter denom_unit to islm (to lowercase)

Reply:
Fixed, thank you ❤️